src/kakarot/accounts/eoa/library.cairo

```diff
@@ -161,9 +163,8 @@ namespace ExternallyOwnedAccount {
 
         // See Argent account
         // https://github.com/argentlabs/argent-contracts-starknet/blob/c6d3ee5e05f0f4b8a5c707b4094446c3bc822427/contracts/account/ArgentAccount.cairo#L132
-        // Using here the EVM msg_hash instead of the Starknet tx.hash
         transaction_executed.emit(
-            msg_hash=msg_hash, response_len=return_data_len, response=return_data, success=success
+            response_len=return_data_len, response=return_data, success=success
```

Comment:
why did you remove the hash btw? it'd be necessary in case of a multicall

Reply:
it's not clear to me how we can leverage multicalls for now:
- we have only access to one signature now since it's in the tx_info (as opposed to one signature per call previously)
- `validate` isn't able to properly validate multiple signed data, so we would have to either:  
    - add many signatures in tx_info.signature field ([r,s,v, r,s,v, ..., r,s,v]), 1 sig per call, and `validate` would validate by jumping from 5 index to 5 index (very clunky imo)
    - sign once the array of calls (better), but we currently don't support this in wallets